wye/social/utils.py

```diff
@@ -0,0 +1,22 @@
+def get_message(context):
+    workshop = context.get('workshop', None)
+
+    if workshop:
+        return construct_message(workshop, context)
+    else:
+        # Other messages for non-workshop creation actions
+        return None
+
+
+def construct_message(workshop, context):
+    date = workshop.expected_date
+    topic = workshop.workshop_section
+    organization = workshop.requester
+    workshop_url = context.get('workshop_url', None)
+    message = "{} workshop at {} on {} confirmed! Details at {}".format(
+        topic, organization, date, workshop_url)
+    if len(message) >= 140:
+        message = "{} workshop on {} confirmed! Details at {}".format(
+            topic, date, workshop_url)
+
+    return message
```

Comment:
@harisibrahimkv can we have a unit test for this method. I'm not very confident that length limit of `140` will always be followed. Primarily due to the variation of `url` length and `topic`.

And also, I think it would make sense to name this method more clearly. Something like `get_workshop_confirmation_tweet` and put that in the `workshops.services` module. Thoughts?